### PR TITLE
Shop disabled button tooltips

### PR DESCRIPTION
**Pull request by ryanprice**
Added a padlock icon to the disabled buttons. I added a tooltip, but would like the tooltip hover on the entire disabled button

### CODE REVIEW

File: src/components/OwnedCardContent.vue [hunk 1]
```diff
@@ -86,7 +86,21 @@
 export default {
   name: 'OwnedCardContent',
   props: ['id','type_id','name','image','edition_total','cset','unlock_czxp','level','cost','buy_czxp','transfer_czxp','sacrifice_czxp','card_class', 'in_store'],
-    computed: {
+  computed: {
+    buyBtnTooltipText() {
+      if (!this.buybutton) {
+        return this.buyBtnTooltipTextContent
+      } else {
+        return '${this.buyBtnTooltipTextContent} changed'
```
Comment (jjjhill) [inline src/components/OwnedCardContent.vue:94]: this works? i thought it had to be \`${backticks}\`

**Review by jjjhill — APPROVED** [review]

File: src/components/OwnedCardContent.vue (hunk 1 above)
Comment (ryanprice) [reply src/components/OwnedCardContent.vue:94]: no you are right, and this code is still broken.. fix coming up soon

Comment (ryanprice) [reply src/components/OwnedCardContent.vue]: Pull and try again

**Review by mchigit — APPROVED** [review]
LGTM

**Pull request merged**